virttest/libvirt_vm.py

```diff
@@ -459,7 +459,8 @@ def add_hvm_or_pv(help_text, hvm_or_pv):
                 logging.warning("Unknown virt type hvm_or_pv, using default.")
                 return ""
 
-        def add_mem(help_text, mem, maxmem=None, hugepage=False):
+        def add_mem(help_text, mem, maxmem=None, hugepage=False,
+                    hotplugmaxmem=None, hotplugmemslots=1):
```

Comment:
why do we have to set `hotplugmemslot` default as 1 ?, it can be `None`

Reply:
`hotplugmaxmem` and `hotplugmemslots` are related, just to make sure the default value incase user not set `hotplugmemslots` explicitly, not sure of default slot value, let me check and set it accordingly.

Reply:
@balamuruhans it expects both, so default as `1` in safest option for now, ```
Starting install...
ERROR    XML error: both maximum memory size and memory slot count must be specified
Domain installation does not appear to have been successful.
```